test/onigumo_downloader_test.exs

```diff
@@ -94,14 +94,13 @@ defmodule OnigumoDownloaderTest do
   end
 
   describe("Onigumo.Downloader.load_urls/1") do
-    @tag :tmp_dir
-    test("load URLs from a file", %{tmp_dir: tmp_dir}) do
-      input_path_env = Application.get_env(:onigumo, :input_path)
-      input_path_tmp = Path.join(tmp_dir, input_path_env)
-
-      for slice <- @slices do
-        input_urls = Enum.slice(@urls, slice)
+    for slice <- @slices do
+      @tag :tmp_dir
+      test("load URLs #{inspect(slice)} from a file", %{tmp_dir: tmp_dir}) do
+        input_urls = Enum.slice(@urls, unquote(Macro.escape(slice)))
```

Comment:
How works `unquote` in combination with `Macro.escape`:

`unquote` is macro, which wrap `Macro.escape` (escape is function of module Macro, module which contain helpful functions for working with macros).

`unquote` put content of variable slice to `Macro.escape` function, we embed range 0..1 or 0..-1 instead of name of variable which will not be defined in frame of test function after compilation process. Finally `Macro.escape` creates AST for complication data structure as `range`.

`unquote` can be used without `Macro.escape` just for simple data structure like integer, string, ......